src/Components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Navbar, Nav } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import 'bootstrap/dist/css/bootstrap.css';
+import './NavBar.scss';
+import { toggleSideNav } from '../../actions/toggleSideNav';
+import Logo from '../Common/Logo/Logo';
+import MiniAccount from '../Common/MiniAccount/MiniAccount';
+import NotifIcon from '../Common/NotifIcon/NotifIcon';
+import SearchBox from '../Common/SearchBox/SearchBox';
+import SideBar from '../SideBar/SideBar';
+
+/**
+ * Render the NavBar
+ * @class NavBar
+ * @extends {Component}
+ */
+export class NavBar extends Component {
+  /**
+   * Creates an instance of NavBar
+   * @param {*} props
+   * @memberof NavBar
+   */
+  constructor(props) {
+    super(props);
+    const { component, onToggleSideNav, currentUser } = props;
+    this.component = component;
+    this.onToggleSideNav = onToggleSideNav;
+    this.currentUser = currentUser;
+  }
+
+  state = {};
+
+  /**
+   * @memberof NavBar
+   * @returns {void}
+   */
+  toggleSideNav() {
+    const { isDrawerDisplay } = this.component;
+    this.onToggleSideNav(isDrawerDisplay);
+  }
+
+  /**
+   * @returns {void}
+   */
+  toggleNotification() {
+    return 'this should toggle notification dropdown';
+  }
+
+  /**
+   * @param {*} isAuth
+   * @param {*} notificationsCount
+   * @param {*} username
+   * @param {*} image
+   * @param {*} user
+   * @returns {object} jsx
+   */
+  renderNavBar(isAuth, notificationsCount, username, image, user) {
+    return (
+      <React.Fragment>
+        <Navbar fixed="top" collapseOnSelect expand="lg" variant="dark">
+          <Navbar.Brand>
+            <Logo />
+          </Navbar.Brand>
+          <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+          <Navbar.Collapse id="responsive-navbar-nav">
+            <Nav className="mr-auto search-container">
+              <SearchBox />
+            </Nav>
+            {!isAuth ? null : (
+              <Nav>
+                <NotifIcon
+                  notificationsCount={notificationsCount}
+                  onClick={this.toggleNotification}
+                />
+                <MiniAccount username={username} avatarImg={image} />
+              </Nav>
+            )}
+          </Navbar.Collapse>
+        </Navbar>
+        <SideBar user={user} />
```

Comment:
@coolbeatz71 Why is the SideBar included in the NavBar?

Reply:
> @coolbeatz71 Why is the SideBar included in the NavBar?

Just for the purpose of viewing how they can display when they get together. If you really need me to remove it, then let me do it